domain/src/main/kotlin/no/nav/su/se/bakover/domain/SøknadInnholdTestdataBuilder.kt

```diff
@@ -5,6 +5,12 @@ import java.time.Month.FEBRUARY
 import java.time.Month.JANUARY
 import java.time.Month.JULY
 
+fun fnrUnder67(): Fnr {
```

Comment:
Kanskje ikke så viktig for denne PRen, men kunne vært interessant å sette opp en test som krever en samboer over 67 også, og sjekke hvordan det vil påvirke denne.